test/e2e/cli.go

```diff
@@ -75,6 +87,52 @@ func (e TknRunner) RunWithOption(option icmd.CmdOp, args ...string) *icmd.Result
 	return icmd.RunCmd(icmd.Cmd{Command: cmd, Timeout: tknTimeout}, option)
 }
 
+// RunInteractiveTests helps to run interactive tests.
+func (e TknRunner) RunInteractiveTests(t *testing.T, ops *Prompt) *expect.Console {
+	t.Helper()
+	// Multiplex output to a buffer as well for the raw bytes.
+	buf := new(bytes.Buffer)
+	c, state, err := vt10x.NewVT10XConsole(goexpect.WithStdout(buf))
+	assert.NilError(t, err)
+	defer c.Close()
+
+	if e.namespace != "" {
+		ops.CmdArgs = append(ops.CmdArgs, "--namespace", e.namespace)
+	}
+
+	cmd := exec.Command(e.path, ops.CmdArgs[0:len(ops.CmdArgs)]...) //nolint:gosec
+	cmd.Stdin = c.Tty()
+	cmd.Stdout = c.Tty()
+	cmd.Stderr = c.Tty()
+
+	err = cmd.Start()
+	if err != nil {
+		t.Fatal(err)
```

Comment:
It might be nice to make this configurable to allow for errors in the event we want to test for errors with interactive tests. Perhaps a param for the func that allows for error checking or fails the test based on an error?